Sources/Liquid/Filter+Standard.swift

```diff
@@ -491,19 +491,20 @@ enum Filters {
 		}
 	}
 	
-	static func dateFilter(value: Value, args: [Value], kwargs: [String: Value], encoder: Encoder) throws -> Value {
+	static func dateFilter(value: Value, args: [Value], kwargs: [String: Value], context: FilterContext) throws -> Value {
 		guard args.count == 1 else {
 			throw RuntimeError.invalidArgCount(expected: 1, received: args.count, tag: tagName())
 		}
 		
-		guard let date = convertValueToDate(value, encoder: encoder) else {
+		guard let date = convertValueToDate(value, context: context) else {
 			throw RuntimeError.wrongType("Could not convert to date")
 		}
 		
 		let format = args[0].toString()
 		let formatter: DateFormatter
 		if format.isEmpty {
 			formatter = DateFormatter()
+			formatter.locale = context.encoder.locale
```

Comment:
forgot to set this, which was causing the tests to fail when not running against an `en-US` environment